File: plot_foo.jl
println("hello, julia")
function fibo(n)
	a, b = 0, 1
	f = []
	while b <= n
		print(b, ' ')
		push!(f, b)
		a, b = b, a+b
	end
	println()
	return f
end

y1 = fibo(60)
println("add a number")
push!(y1, 100)
println(y1)

using Plots
pyplot()
x = 1:11
p1 = plot(x,y1)
y2 = rand(11)
p2 = scatter(x,y2)
display(p1)

File: zzz/plot_foo.py
#!/usr/bin/env python3
# -*- coding:utf-8 -*-
def fibo(n):
    print("hello, python")
    f = []
    a, b = 0, 1
    while b <= n:
        print(b, end=' ')
        f.append(b)
        a, b = b, a+b
    print()
    return f

y1 = fibo(60)
print("add a number")
y1.append(100)
print(y1)

import numpy as np
import matplotlib.pyplot as plt
x = np.linspace(1,11,11)
plt.figure()
plt.plot(y1)

y2 = np.random.random(11)
plt.scatter(x,y2,color="red")
plt.show()
